lib/libsysfs.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=#libdir@
```

Comment:
Thanks for improving the build, a minor nit-pick `libdir=#libdir@` should be `libdir=@libdir@`

Reply:
Ah! Thanks, fixed now, seems I missed that when checking the generated output. :)